packages/app/src/runner/screenshot/screenshot.cy.tsx

```diff
@@ -45,7 +55,9 @@ describe('screenshot', () => {
         styles,
       })
 
-      cy.screenshot(`percy/component_testing_takes_a_screenshot_viewport_${capture}`, { capture })
+      removeGlobalStyles().then(() => {
+        cy.screenshot(`percy/component_testing_takes_a_screenshot_viewport_${capture}`, { capture })
+      })
```

Comment:
```suggestion
      removeGlobalStyles()
      cy.screenshot(`percy/component_testing_takes_a_screenshot_viewport_${capture}`, { capture })
```

AFAIK you don't need `.then` because Cypress runs promises sequentially and you're not using the return value from `removeGlobalStyles` within `cy.screenshot`